app/templates/login.hbs

```diff
@@ -1,5 +1,15 @@
-<div class="ui one column container stackable doubling centered grid">
+<div class="ui one column container stackable doubling centered grid text-center">
   <div class="column">
-    <Forms::LoginForm />
+    {{#if (not-eq this.session.Passage "/login")}}
+      {{#if (or (eq this.session.previousRouteName '/') (eq this.session.previousRouteName '/create') (eq this.session.Passage '1'))}}
```

Comment:
Will you add hundreds of cases here like this?

Reply:
No, but if we need different alert message for different case then I have to make cases for them.

Reply:
I will surely try to generalize the cases.

Reply:
can I use a action method and check every time that which button is pressed and show the message respectively.

Reply:
No, that is too much work

Reply:
when we verify our email address it automatically login ,So should I change the message which appears on the screen or should I prevent it from login automatically and show that message.

Reply:
The issue is about messages shown on login screen when you are redirected to them